src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -60,7 +64,14 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
     // the default selection handler of the entity, which the field reference
     // to, and add another logic to the query object i.e. check if the entities
     // bundle defined as group.
-    $query = $this->getSelectionHandler()->buildEntityQuery();
+    $selection_handler = $this->getSelectionHandler();
+
+    if (empty($selection_handler)) {
+      return $query = parent::buildEntityQuery($match, $match_operator);
```

Comment:
I think so, as the field could not always be configured, like when it is called for validation constraints. See the `ValidReferenceContraintValidator`. If there is not a field, there is no way the `OgSelection` handler can get the base handler. Other handler don't work this way or rely on this value, so it's not a problem.